subsys/settings/Kconfig

```diff
@@ -124,3 +124,9 @@ config SETTINGS_NVS_SECTOR_COUNT
 	depends on SETTINGS && SETTINGS_NVS
 	help
 	  Number of sectors used for the NVS settings area
+
+config SETTINGS_SHELL
+	bool "Enable Settings shell"
+	depends on SETTINGS && SHELL
+	help
+	  Enable shell commands for listing and reading the settings.
```

Comment:
Since the SETTINGS_MAX_VAL_LEN is quite large, 256 by default, consider adding note here that enabling this shell feature impacts stack usage and may require altering stack size.

Reply:
Ok, done.